activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Add `zettabytes` method.
+    Add alias method `zettabyte` to `zettabytes`.
+    Add `yottabytes` method.
+    Add alias method `yottabyte` to `yottabytes`.
```

Comment:
Since only listed `zettabytes` and `yottabytes` in the Bytes section in the Active Support Core Extensions guide, it is enough to list only `zettabytes` and `yottabytes`.

```
*   Add `zettabytes` and `yottabytes` methods.
```